src/main/plugins/org.dita.pdf2/xsl/fo/commons.xsl

```diff
@@ -459,6 +459,9 @@ See the accompanying license.txt file for applicable licenses.
 
     <xsl:template name="processTopicNotices">
         <fo:page-sequence master-reference="body-sequence" xsl:use-attribute-sets="page-sequence.notice">
+            <xsl:if test="key('map-id', ancestor-or-self::*[contains(@class, ' topic/topic ')][1]/@id)/ancestor::*[contains(@class,' bookmap/backmatter ')]">
```

Comment:
Ah, I was assuming two attribute sets - so anybody who has overridden the current one would be good, but we'd still have an attribute set that lets people override the back matter version. Something like:
```
<xsl:choose>
  <xsl:when test="back matter notices">
    <fo:page-sequence master-reference="body-sequence" xsl:use-attribute-sets="page-sequence.notice.backmatter">
      ...call template for everything else...
    </fo:page-sequence>
  </xsl:when>
   <xsl:otherwise>
     <fo:page-sequence master-reference="body-sequence" xsl:use-attribute-sets="page-sequence.notice">
      ...call template for everything else...
    </fo:page-sequence>
  </xsl:when>
</xsl:choose>
```

Advantage there is that no current customizations would be affected, because the front matter attribute set is untouched. We still fix the back matter one, but it can also be updated via attribute set. The only code duplication is that we have two `<fo:page-sequence>` elements, with different attribute sets.